smithy-model/src/test/resources/software/amazon/smithy/model/transform/dedicated-input-output/handles-dangling-single-reference-before.smithy

```diff
@@ -0,0 +1,13 @@
+$version: "2.0"
+
+namespace smithy.example
+
+// We have a shape being used by both input and output
+// so there will be an extra or "dangling" reference
+// to `GetFooOutput` from the input
+operation GetFoo {
+    input: GetFooOutput
+    output: GetFooOutput
+}
+
+structure GetFooOutput {}
```

Comment:
Can the naming of this output structure get updated so it's clear that it will be removed in the `-after.smithy` file?

Reply:
I added a second test that verifies that the unused shape will be removed, I'll leave this open to make sure that looks ok